routes/service.php

```diff
@@ -54,6 +54,17 @@
         'as' => 'admin.vouchers.retirebatch',
         'uses' => 'Admin\VouchersController@retireBatch',
     ]);
+    Route::get('vouchers/viewone/{voucher}', [
+        'as' => 'service.vouchers.viewone',
+        'uses' => 'Admin\VouchersController@viewOne',
+    ])->where('voucher', '^[0-9]+$');
+
+    Route::view('/service/vouchers/edit', 'edit');
```

Comment:
unusual - I would expect `Route::get('vouchers/{id}/edit')`

is that already a route somewhere?

Reply:
It turns out I didn't even need that route, so it's gone away.